certified-connectors/AssemblyAI/apiDefinition.swagger.json

```diff
@@ -1361,12 +1529,122 @@
         "language_code": {
```

Comment:
Kindly confirm that the existing flows will not be broken from the below swagger validation error. 

The list of enum values has been added to the input field. Existing flows may use values outside of this list. ValuePath: operation/CreateTranscript/parameter/body/schema/language_code/enum

Reply:
If they already used a custom value, they would continue to use that custom value, right?
If the custom value is the same as a new enum value, that wouldn't be a problem either, right?
How can we verify this?

Reply:
Ok, as you mentioned that this connector has not been released yet, we will consider this change.